e2e/cases/dts/index.test.ts

```diff
@@ -1,20 +1,76 @@
 import { join } from 'node:path';
 import { buildAndGetResults } from '@e2e/helper';
-import { expect, test } from 'vitest';
+import { describe, expect, test } from 'vitest';
 
-test('dts when bundle: false', async () => {
-  const fixturePath = join(__dirname, 'bundle-false');
-  const { files, contents } = await buildAndGetResults(fixturePath, 'dts');
+describe('dts when bundle: false', () => {
+  test('basic - bundleless dts', async () => {
```

Comment:
We could use nested `describe` instead of manual `basic -` prefix, and it will be easier to test with filter.

Reply:
Snapshot cannot be used outside of test, nested describe is not a good choice. I think we can simply remove `- xxx`

![image](https://github.com/user-attachments/assets/43fadcef-e358-4452-94f2-6a28c7022d60)
 
The log is clear enough